client/src/components/ui/LogInModal.tsx

```diff
@@ -0,0 +1,73 @@
+import Image from "next/image";
+
+import { Button } from "@/components/ui/button";
+import {
+  Dialog,
+  DialogContent,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+} from "@/components/ui/dialog";
+import { Input } from "@/components/ui/input";
+import { Label } from "@/components/ui/label";
+
+interface Props {
+  isOpen: boolean;
+  onClose: () => void;
+}
+
+function LogInModal({ isOpen, onClose }: Props) {
+  return (
+    <Dialog open={isOpen} onOpenChange={onClose}>
+      <DialogTitle asChild />
+
+      <DialogContent
+        className="flex h-full max-h-[460px] w-[95%] max-w-[600px] flex-col items-center overflow-y-auto rounded-[40px] border-0 bg-[#9DAD93] p-1 shadow-lg file:mx-auto"
+        style={{ backgroundColor: "#9DAD93", borderRadius: "32px" }}
+      >
+        <div className="w-full rounded-[30px] border-4 border-[#9DAD93] bg-white p-4">
+          {/*Image */}
+          <div className="w-30 h-30 mb-1 flex justify-center">
+            <Image
+              src="/web_logo.png"
+              alt="CO-EXIST LOGO"
+              width={150}
+              height={150}
+            />
+          </div>
+          <div className="mb-2 flex justify-center border-b-2 border-[#9DAD93]">
+            <DialogHeader>
+              <DialogTitle className="text-xl font-semibold">Login</DialogTitle>
+            </DialogHeader>
+          </div>
+
+          {/* Labels */}
+          <div className="grid gap-4 py-4">
+            <div className="gap-2">
```

Comment:
This "gap-2" class is not doing anything since it it not in a flex box. This can be removed. Same with the password field. The First and Last Name inputs do have working gaps since they are using cols, however if gaps could be removed from those as well so to remain consistent with the other labels that would be good.